bootcamp_module08/contrib/student_03.py

```diff
@@ -0,0 +1,18 @@
+def reverse_string(string):
+    """Reverses a string
+
+    Parameters
+    ----------
+    string : str
+        The string to reverse
+
+    Returns
+    -------
+    reversed_string: str
+        the 'string' with all characters in reversed order
+
+    """
+    reversed_string = ""
+    for ii in range(len(string)-1, -1, -1):

```

Comment:
```suggestion
    for ii in range(len(string) - 1, -1, -1):
```

I might also rename this variable from `ii` to something more informative.